eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/BinlogEntryReader.java

```diff
@@ -117,6 +122,7 @@ protected void onActivity() {
 
   protected void handleProcessingFailException(Exception e) {
     logger.error(e.getMessage(), e);
```

Comment:
Add a useful message rathe than duplicate the exception's message `logger.error("Stopping due to exception", e)`